src/common/GroupAvatar.js

```diff
@@ -53,17 +53,17 @@ export default class GroupAvatar extends PureComponent<Props> {
       height: size,
       width: size,
       borderRadius: size / 8,
-      backgroundColor: colorHashFromString(names.join(', ')),
-    };
-    const textSize = {
-      fontSize: size / 3,
-      color: foregroundColorFromBackground(frameSize.backgroundColor),
+      backgroundColor: Color(colorHashFromString(names.join(', ')))
+        .lighten(0.6)
+        .hex(),
     };
 
+    const iconColor = Color(colorHashFromString(names.join(', '))).string();
+
     return (
       <Touchable onPress={onPress}>
         <View style={[styles.frame, frameSize]}>
-          <Text style={[styles.text, textSize]}>{initialsForGroupIcon(names)}</Text>
```

Comment:
Ah, sorry I missed this until now: it looks like this was the only place where we were using the `initialsForGroupIcon` function, so it's now dead code. So let's remove it, and also remove src/common/__tests__/initialsForGroupIcon-test.js. Those removals should both happen in the same commit as each other; could be squashed with the existing commit or could be a new commit after it.

Reply:
Sure, will remove it, even I also missed `initialsForGroupIcon` function 😅. I will do both removal of the `initialsForGroupIcon` function and src/common/tests/initialsForGroupIcon-test.js in a single commit and then I will squash both the commits into a single commit. Squashing would be fine because, this is a not a very huge change and also single commit would be easy to understand 🙂.